### PR TITLE
Genesys Cloud container update  and Intermediate Transcript support in Pub/sub interceptor

### PR DESCRIPTION
**Pull request by carinali100**
Genesys Cloud and Intermediate Transcript support in Pub/sub interceptor. Updated the Genesys Cloud main.ejs
point to container.js v1.11. Update the cloud-pubsub-interceptor to process the new event for intermediate Transcript

Tested: Both the backend module and frontend module are redeployed with the code change in the project
Live Transcript: https://screencast.googleplex.com/cast/NTkyMzQ1NDAzNjQ3NTkwNHw2ODkwZTQ2Mi1jYQ
intermediate Transcript: https://screencast.googleplex.com/cast/NTQ4ODkwNDYxMzM5NjQ4MHxjNzhkMzE4NC0yMw
chat: https://screenshot.googleplex.com/9XPeZWt3JKCp5UW

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 *.code-workspace
+.env
+commit.txt
```
Comment (deliaqi) [inline .gitignore]: nit: what is this file for?

**Review by deliaqi — APPROVED** [review]
Mostly LGTM, thanks!

File: .gitignore (hunk 1 above)
Comment (carinali100) [reply .gitignore]: it is the commit.txt message file I used for the command `git commit -F commit.txt`

Comment (carinali100) [reply .gitignore]: deleted commit.txt from .gitignore

**Pull request merged**